packages/backend-common/src/tokens/ServerTokenManager.ts

```diff
@@ -26,40 +26,47 @@ import { TokenManager } from './types';
  * @public
  */
 export class ServerTokenManager implements TokenManager {
```

Comment:
Todo: split out the noop token manager into a separate class.

Reply:
Done in 10d0109d2e.

Reply:
Moved inside the `ServerTokenManager` namespace in 54eaeddaf1, feedback welcome :)